hazelcast-spring/src/main/resources/hazelcast-spring-3.9.xsd

```diff
@@ -2427,6 +2427,7 @@
             <xs:element name="ssl" type="ssl" minOccurs="0" maxOccurs="1"/>
             <xs:element name="aws" type="aws-client" minOccurs="0" maxOccurs="1"/>
             <xs:element name="discovery-strategies" type="discovery-strategies" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="outbound-ports" type="outbound-ports" minOccurs="0" maxOccurs="1"/>
```

Comment:
@asimarslan: the @gokhanoner's explanation makes sense to me. having this aligned with member-side is nice, isn't it?